integration_test/third_party_apps_data/applications/varnish/metadata.yaml

```diff
@@ -1,5 +1,6 @@
 short_name: varnish
 long_name: Varnish HTTP Cache
```

Comment:
It also needs Description from the scope doc, it will start failing the test once https://github.com/GoogleCloudPlatform/ops-agent/pull/567 is in

```
The Varnish integration collects cache and session metrics. It monitors the number of objects entering and exiting the cache, as well as the number of sessions and backend connections. The integration also collects Varnish access logs and parses them into a standardized json payload.
```

Reply:
Added